src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -186,11 +194,18 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
   /**
     * Divide this block into two child blocks if a joint intersects this block.
     * @param joint A joint that may or may not divide this block.
+    *  @param minSize The minimum radius of a sphere that can be inscribed in the child blocks.
+    *                 If either child block falls below this minimum, no cut is performed.
+    *  @param maxAspectRatio The maximum ratio of a child block's bounding sphere to the radius
+    *                        of the largest sphere that can be inscribed in the block. If either
+    *                        child falls above this minimum, no cut is performed.
     * @return A Seq of Block objects, containing the two child blocks divided by
-    * the joint if it intersects this block. Otherwise, returns a one-item Seq
-    * containing only this block.
+    * the joint if it intersects this block and any minimum requirements for radius
+    * or aspect ratio are met. Otherwise, returns a one-item Se containing
+    * only this block.
     */
-  def cut(joint: Joint): Seq[Block] = {
+  def cut(joint: Joint, minSize: Double=Double.NegativeInfinity,
```

Comment:
Sure. I verified that we won't into any issues if we use `Double.NegativeInfinity`, but I'll switch it to `0.0` since that probably makes more sense anyways.